community/lexicon/bookmarks/getActorBookmarks.json

```diff
@@ -0,0 +1,66 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.bookmarks.getActorBookmarks",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a list of bookmarks by actor. Optionally add a list of tags to include, default will be all bookmarks. Requires auth, actor must be the requesting account.",
```

Comment:
Requiring auth and also the actor param where it will always be self seems redundant. Maybe remove the actor param?